table_connector.py

```diff
@@ -257,3 +258,14 @@ def get_cleaned_fully_merged_messages():
     print 'Printing first row of address book with full name column (merged first name/last name/company):'
     display(address_book_df.head(1))
     return fully_merged_messages_df, address_book_df
+
+if __name__=="__main__":
+    messages, addresses = get_cleaned_fully_merged_messages()
+    if len(sys.arv)>1:
+        file_path = sys.argv[1]
+        messages.to_csv(file_path+"messages.csv")
+        addresses.to_csv(file_path+"addresses.csv")
+    else:
+        print messages
```

Comment:
Instead of leaning on the default print, why not just write out the csv here?

You can do something like:

```
from StringIO import StringIO
buffer = StringIO()
messages.to_csv(buffer, index=None, na_rep='None')
print buffer.getvalue()
```